models/dataSource.js

```diff
@@ -0,0 +1,22 @@
+const { DataSource } = require('typeorm');
+
+const appDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+appDataSource
+    .initialize()
+    .then(() => {
+        console.log('Data Source has been initialized!');
+    })
+    .catch((err) => {
+      console.error('Error occurred during Data Source initialization', err);
+     DataSource.destroy();
```

Comment:
새로이 만들어진 DataSource 인스턴스는 appDataSource 라는 변수에 담아주었습니다. 그렇기 때문에 import 해온 DataSource를 destroy하는 것이 아닌 만들어진 appDataSource를 해주셔야 합니다!
```suggestion
     appDataSource.destroy();
```